proto/hummock.proto

```diff
@@ -310,6 +310,7 @@ message CompactTask {
   bool split_by_state_table = 21;
   // Compaction needs to cut the state table every time 1/weight of vnodes in the table have been processed.
   uint32 split_weight_by_vnode = 22;
```

Comment:
Are they deprecated now?

Reply:
No in this PR, the new `table_vnode_partition` is not persistent